src/common/SearchEmptyState.js

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ZulipButton, Label } from './';
+
+const styles = StyleSheet.create({
```

Comment:
I think `componentStyles` is being used instead of `styles` in order to avoid name conflict with `styles` that's being fetched from context e.g. https://github.com/zulip/zulip-mobile/blob/master/src/start/PasswordAuthView.js#L84. Pinging @borisyankov for a double check :)